### ISSUE TITLE
golang layer "test current function"

### ISSUE THREAD

**Issue by aaronkavlie-wf** [syl20bnr/spacemacs#4715]
The go layer ability to run the current test function (`SPC m t t`) does not seem to function properly. I tried multiple times in a `_test.go` file, with the cursor placed in a test function, and got the error `Search failed: [long regex]`.

**Comment by TheBB** [comment]
@bogdanteleaga

**Comment by bogdanteleaga** [comment]
So I tried this right now and it does seem to still work. What version are you using?

Also can you point me to the file you're trying this in (in case it's freely available) ?
[This](https://github.com/syl20bnr/spacemacs/blob/master/layers/%2Blang/go/packages.el#L45) is probably the root cause but the length of the regex doesn't change from one system to another so I'm rather curious why you're getting long regex.

**Comment by newhook** [comment]
It doesn't work for me either. I get:

```
save-excursion: Search failed: "^func[ ]+([[:alnum:]]*?[ ]?[*]?\\([[:alnum:]]+\\))[ ]+\\(Test[[:alnum:]]+\\)(.*)"
```

The current function is:
```
func TestState(t *testing.T) {
```

**Comment by aaronkavlie-wf** [comment]
I am on `go-mode-20151226.1224`, Spacemacs 0.105.9

In my description, [long regex] was just a placeholder for the regex, as I don't know how to copy-paste errors from the status line. The line you linked to looks like it matches the regex in the error message I get.

**Comment by bogdanteleaga** [comment]
Just opened a PR, could you test that code?

**Comment by aaronkavlie-wf** [comment]
I just run on official releases. What steps should I take to test the code?

**Comment by bogdanteleaga** [comment]
Take a look [here](https://github.com/syl20bnr/spacemacs/pull/4732/files) and replace the red lines with the green ones. You can see the file in the header.

**Comment by aaronkavlie-wf** [comment]
OK, manually copy-pasted the code changes, and it works for me. Thanks!

**Comment by newhook** [comment]
@aaronkavlie-wf For future reference you can copy the details from *Messages*.

**Comment by aaronkavlie-wf** [comment]
@newhook took me a second to understand... ok, the buffer `*Messages*`. Cool, thanks for the tip!

**Comment by newhook** [comment]
Any chance this can be fixed on the mainline? I keep updating spacemacs and wiping out my local changes...

**Comment by TheBB** [comment]
This is not a very high priority bug when it comes to hotfixing. If this is important for you, is there any reason you can't just switch to the development branch for a while?

**Comment by syl20bnr** [comment]
Will in next update 0.105.14
No ETA for it though.

**Comment by newhook** [comment]
I don't know about the spacemacs development methodology but I'd expect a fix (especially simple fixes like this) which corrects documented functionality to be merged promptly.

**Comment by syl20bnr** [comment]
Released in 0.105.14

@newhook It was merged in develop but not back ported to master.
